API.md

```diff
@@ -589,3 +589,164 @@ The response will be the requested data as FHIR Patient resources in NDJSON form
 	}
 }
 ```
+
+### Beneficiary Coverage Data
+	The process of retrieving coverage data is very similar to all of the other exporting operations supported by this API.
+
+#### 1. Obtain an access token
+	See [Authentication and Authorization](#authentication-and-authorization) above.
+
+#### 2. Initiate an export job
+
+##### Request
+	`GET /api/v1/Coverage/$export`
+
+To start a coverage data export job, a GET request is made to the Coverage export endpoint. An access token as well as `Accept` and `Prefer` headers are required.
+
+The dollar sign ('$') before the word "export" in the URL indicates that the endpoint is an action rather than a resource. The format is defined by the [FHIR Bulk Data Export spec](https://github.com/smart-on-fhir/fhir-bulk-data-docs/blob/master/export.md).
+
+###### Headers
+	* `Authorization: Bearer {token}`
+	* `Accept: application/fhir+json`
+	* `Prefer: respond-async`
+
+###### cURL command
+	```sh
+	curl -v https://sandbox.bcda.cms.gov/api/v1/Coverage/\$export \
+	-H 'Authorization: Bearer {token}' \
+	-H 'Accept: application/fhir+json' \
+	-H 'Prefer: respond-async'
+	```
+
+  ##### Response
+	If the request was successful, a `202 Accepted` response code will be returned and the response will include a `Content-Location` header. The value of this header indicates the location to check for job status and outcome. In the example header below, the number 43 in the URL represents the ID of the export job.
+
+  ###### Headers
+	* `Content-Location: https://sandbox.bcda.cms.gov/api/v1/jobs/43`
```

Comment:
Should we use a different job ID for this example? Even though they're meant to be standalone, the other two examples have different IDs (42, 43), so to be on the safe side maybe we want to use 44 or some other number for this one.

Reply:
good point, will do! and good catch